Source/DIConnect/ClientApp/public/locales/en-US/translation.json

```diff
@@ -42,15 +42,15 @@
   "SendToRosters": "Send in chat to members of the following teams",
   "SentToRosters": "Sent in chat to people in teams",
   "SendToRostersPlaceHolder": "Choose team(s) members",
-  "SendToGroups": "Send in chat to members of the following M365 groups, Distribution groups or Security groups",
+  "SendToGroups": "Send in chat to members of the group(s) you specify (E.g., M365 group, Distribution group, or Security group.)",
```

Comment:
e.g.
(all lower case)
Better yet, "for example"
Also no period at the end of "Security group"

Reply:
updated